docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -36,40 +53,101 @@ Start by fetching Chipyard's sources. Run:
     # checkout latest official chipyard release
     # note: this may not be the latest release if the documentation version != "stable"
     git checkout |version|
+
+If you are running Chipyard alongside FireSim on AWS EC2, you should skip the :ref:`Chipyard-Basics/Initial-Repo-Setup:Conda Environment Setup` section and instead jump to :ref:`Chipyard-Basics/Initial-Repo-Setup:Fetch Chipyard Sources`.
+
+Conda Environment Setup
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. Warning:: When running on an Amazon Web Services EC2 FPGA-development instance
+    (for FireSim), FireSim includes a similar machine setup script that will install all
+    of the aforementioned dependencies (and some additional ones) and will activate the
+    proper conda environment. Skip this section.
+
+Next run the following command to create Chipyard's Conda environment.
+
+.. code-block:: shell
+
+    conda env create -f scripts/conda-requirements.yml
```

Comment:
```suggestion
    conda env create -f scripts/conda-requirements.yaml -n chipyard
```

The file extension should be `.yaml`.
In my case, the `-n` option turned out to be mandatory.